src/js/otus/hooks.ts

```diff
@@ -0,0 +1,35 @@
+import { useFetchOTU } from "@otus/queries";
+import { OTUSequence } from "@otus/types";
+import { indexOf, map, sortBy } from "lodash-es/lodash";
+
+/**
+ * A hook for sorting the sequences for the active isolate
+ *
+ * @param otuId - The otu which the isolate belongs to
+ * @param sequences - The sequences in the active isolate
+ */
+export default function useGetSequences(otuId: string, sequences: OTUSequence) {
+    const { data, isLoading } = useFetchOTU(otuId);
```

Comment:
I think it's sketchy to use a query and not expose the loading status to the calling component. the calling component has no way of knowing if there is no sequences, or if the query simply isn't cached. For this hook I would go one of two directions:

1. lean into it being a query. Take only the Ids you need and fetch the rest from the API. From the perspective of the calling component the interface of `useGetSequences` matches as closely as possible the interface of a standard query. 

2. Explicitly pass the data via props, context, ect and turn this into a simple util function that does the sorting